src/drivers/imx/CMakeLists.txt

```diff
@@ -3,7 +3,6 @@
 add_local_sources(sof
 	esai.c
 	sai.c
-	interrupt.c
```

Comment:
Do we need to delete this or is it still used ?

Reply:
The file was renamed and moved to be conditional. Currently all i.MX platforms that work have irqsteer enabled so it doesn't matter. @TangleZ said in internal mailing lists that he will send a second pull request to actually add the support proper for 8ULP from this point of view.

I expect a future pull request to add a new interrupt.c file which has some simple, empty functions to satisfy the linker, given that 8ULP only has arch interrupts and the arch_ functions suffice for our use case.